### ISSUE TITLE
Incomplete UIA mappings for aria-haspopup

### ISSUE THREAD

**Issue by sivakusayan** [w3c/core-aam#196]
While investigating browser support for https://github.com/w3c/core-aam/issues/190, I noticed that [Chromium exposes `haspopup`](https://github.com/chromium/chromium/blob/78f6e36bd96ebe7ceb38d754c75e6d92613ebdb7/ui/accessibility/platform/ax_platform_node_win.cc#L7249) as a property under `AriaProperties` in UIA. I don't think I see this called out in any of the [UIA mappings](https://www.w3.org/TR/core-aam-1.2/#ariaHaspopupTrue) in the spec. 

If I'm not misunderstanding something, I think this can be fixed by just adding a `AriaProperties.haspopup: <value>` to each mapping.

**Comment by benbeaudry** [comment]
I agree with you. To go a step further, this lack of explicit call out in Core-AAM appears to be more widespread and is extended to pretty much all aria attributes, not just aria-haspopup*. Only a subset of aria attributes explicitly mention the AriaProperties mapping. It would be nice to have a uniform solution across all aria properties. Could it be a general call out in a subsection? I fear it would be too much clutter to repeat it in every single aria property. What do you think?

If you want to whip up a PR to update that, you'll certainly get my approval!

**Comment by sivakusayan** [comment]
Sorry for the late response - I've been trying to balance my time more between my full-time job and doing this on the side 🙂 

At first, I was wondering if it would be strange for implementors if we called out the mandatory `AriaProperties` mapping in a separate section instead of in the mapping table itself. This is because I feel like the average reader might just jump to the mapping table immediately instead of taking the time to read the surrounding text. However, I notice that we have a [General rules](https://w3c.github.io/core-aam/#roleMappingGeneralRules) section already, so I guess we can just mention `AriaProperties` under that same heading.

**Comment by spectranaut** [comment]
To be honest, I lean toward including a line in every attribute mapping table for the reasons @sivakusayan mentioned... philosophically I lean towards being a pendant as opposed to avoiding repetition in specs anyway -- because in general specs are read more than they are edited.

I'll make a new issue for this :)
